<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="graphWidget" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>571</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="gyroButton">
     <property name="text">
      <string>Giroscopio</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tempButton">
     <property name="text">
      <string>Temperatura</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pressureButton">
     <property name="text">
      <string>Presión</string>
     </property>
    </widget>
    <widget class="QPushButton" name="accButton">
     <property name="text">
      <string>Acelerómetro</string>
     </property>
    </widget>
    <widget class="QPushButton" name="magButton">
     <property name="text">
      <string>Magnetómetro</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sensButton">
     <property name="text">
      <string>Sensor externo</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
